clients/go/datacatalog/client.go

```diff
@@ -0,0 +1,112 @@
+package datacatalog
+
+import (
+	"context"
+	"crypto/x509"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/flyteorg/flyteidl/clients/go/datacatalog/mocks"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/datacatalog"
+
+	"github.com/flyteorg/flytestdlib/logger"
+	grpcRetry "github.com/grpc-ecosystem/go-grpc-middleware/retry"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials"
+)
+
+/*
```

Comment:
so package docs should be the first few lines in the file - or you can add a doc.go